js/Makefile

```diff
@@ -342,14 +359,9 @@ android/libs/gobridge.aar: $(bridge_src)
 		./go/framework/bertybridge
 	touch $@
 
-android/app/debug.keystore:
-	cp node_modules/react-native/template/android/app/debug.keystore $@
-
 #### React
 
-$(pkg_mod): $(pkg_desc)
+node_modules: package.json
```

Comment:
the dependency should be yarn.lock I think

Reply:
The previous rule was only based on `{node_modules,packages/*/nodemodules}: {package.json,packages/*/packages.json}` and it looks ok for me. But I can add `node_modules: package.json yarn.lock` if you want to ;)

Reply:
Didn't that into account that if we only have `yarn.lock`, editing the `package.json` and running the make rules won't run `yarn install`, you're right, I'm for `package.json` only then